crates/bevy_utils/src/short_names.rs

```diff
@@ -1,7 +1,7 @@
 /// Lazily shortens a type name to remove all module paths.
 ///
 /// The short name of a type is its full name as returned by
-/// [`std::any::type_name`], but with the prefix of all paths removed. For
+/// [`core::any::type_name`], but with the prefix of all paths removed. For
```

Comment:
can we keep references to `std` in doc comments or does that trigger the lint?

Reply:
Unfortunately yes, the lint does _not_ catch using `std` in documentation links or examples. As such I will revert changes to documentation since we can't enforce the use anyway.

However, `no_std` crates like `bevy_utils` and `bevy_ptr` _need_ to link to `alloc` and `core` since `std` is unavailable.